projects/1.16/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/exploration/index.json

```
@@ -0,0 +1,4 @@
+{
+  "title": "探索",
+  "subText": "在我的旅行中，我找到了黏液岛的几个变种，它们既是黏液的源产地，也是研究这些有趣的植物群的好地方。此外，其中还有一些独特的资源，但是在我的其书中没有讨论太多，我都想涵盖。I have located several variants of slime islands in my travels, which serve both as breeding grounds for slimes and a source of interesting flora for research. In addition, there are a few unique resources that are not discussed much in my other textbooks that I wanted to cover."
```

```suggestion
  "subText": "在我到处旅行的时候，我找到了黏液岛的几个变种，它们既是黏液的源产地，也是研究这些有趣的植物群的好地方。此外，这些岛上还有一些独特的资源，不过在我想介绍的另外几本书里没有过多涵盖。"
```
这句话什么乱七八糟的，作者知道自己在写啥么